x/concentrated-liquidity/types/genesis/genesis.go

```diff
@@ -27,8 +25,6 @@ func (gs GenesisState) Validate() error {
 	if gs.NextIncentiveRecordId == 0 {
 		return types.InvalidNextIncentiveRecordIdError{NextIncentiveRecordId: gs.NextIncentiveRecordId}
 	}
-	if gs.Params.BalancerSharesRewardDiscount.LT(sdk.ZeroDec()) || gs.Params.BalancerSharesRewardDiscount.GT(sdk.OneDec()) || (gs.Params.BalancerSharesRewardDiscount == sdk.Dec{}) {
-		return types.InvalidDiscountRateError{DiscountRate: gs.Params.BalancerSharesRewardDiscount}
-	}
```

Comment:
This seems like an important check, is this removed bc of defense in depth? If so, can you show where this is being validated?

Reply:
+1

Reply:
This is a param check, that is already being done in params.ValidateBasic() This was a duplicate / redundant check

Reply:
The param validation should be more than sufficient to cover this so I'm in support of removing the check